aspnet/fundamentals/request-features.rst

```diff
@@ -12,63 +12,61 @@ Individual web server features related to how HTTP requests and responses are ha
 Feature interfaces
 ------------------
 
-ASP.NET Core defines a number of `HTTP feature interfaces <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Http/Features/index.html>`_, which are used by servers to identify which features they support. The most basic features of a web server are the ability to handle requests and return responses, as defined by the following feature interfaces:
+ASP.NET Core defines a number of HTTP feature interfaces in the :dn:ns:`Microsoft.AspNetCore.Http.Features` namespace, which are used by servers to identify which features they support. The most basic features of a web server are the ability to handle requests and return responses, as defined by the following feature interfaces:
 
-`IHttpRequestFeature <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Http/Features/IHttpRequestFeature/index.html>`_
+:dn:iface:`~Microsoft.AspNetCore.Http.Features.IHttpRequestFeature`
```

Comment:
It means when producing the link just show the last segment instead of showing the full namespace. See http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-syntax